package.json

```diff
@@ -58,15 +58,19 @@
   },
   "dependencies": {
     "3d-view": "^2.0.0",
-    "@plotly/d3-sankey": "^0.5.1",
     "alpha-shape": "^1.0.0",
     "array-range": "^1.0.1",
     "canvas-fit": "^1.5.0",
     "color-normalize": "^1.3.0",
     "convex-hull": "^1.0.3",
     "country-regex": "^1.1.0",
     "d3": "^3.5.12",
+    "d3-sankey": "git://github.com/antoinerg/d3-sankey.git#4f37ed8d3578b545a8569ecd74583f373768e900",
+    "d3-sankey-circular": "git://github.com/antoinerg/d3-sankey-circular.git#e5760737ef64afade7906bd018149bf1da080e2d",
+    "d3-array": "1",
```

Comment:
Looks like we don't need to list `d3-array`, `d3-collection` and `d3-interpolate` in our deps. This here looks like an artefact from https://github.com/plotly/plotly.js/pull/3355/commits/03a3cb578bc33a06e0c27ed1d91bb79c32e673f4

Reply:
Done in dee73b87c23f4667992718d97251be27385f823b